CHANGELOG.md

```diff
@@ -174,6 +174,12 @@ This project tries to follow [SemVer 2.0.0](https://semver.org/).
   - `test_result_summary.file_name`
   - `token.user_name`
 
+- Added new field `RemoteProjectID` to: (#112)
+
+  - `pkg/model/database.Project`
+  - `pkg/model/request.Project`
+  - `pkg/model/response.project`
```

Comment:
typo project->Project

Also, I think it's fine to just say "the project model"

```suggestion
- Added new field `RemoteProjectID` to the project model. (#112)
```